src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt

```diff
@@ -84,7 +84,7 @@ object AutoEZ : Module() {
             "What happens if I add chocolate milk to macaroni and cheese?",
             "Can you paint with all the colors of the wind"
     ) // Got these from the forums, kinda based -humboldt123 
-    // Got an updated version of them - Natan515
+     // Got an updated version of them - Natan515
```

Comment:
Why is there an extra indent here now?

Reply:
I'm not sure what you're talking about, I just added my own // as the last guy did.

Reply:
There is an extra space though 
![image](https://user-images.githubusercontent.com/38266782/98603225-b79be300-22af-11eb-8c3c-c762f6f18c8a.png)

Reply:
Ok ye but i just removed my line so it should be fine now right?